setup.py

```diff
@@ -31,6 +32,14 @@
         opt_reqs += [ir]
 extras_require['extras'] = opt_reqs
 
+# If interested in benchmarking devito, we need the `examples` too
+exclude = ['docs', 'tests']
```

Comment:
I think we need to add `__init__.py` in these two directories or they are not actually ignore because exclude expects modules not directory.